app/search.py

```diff
@@ -18,8 +19,11 @@ def search_my_term(term, start, count):
     return [url for url in search(term, start=start, stop=count)]
 
 def ignore():
-    with open("data/ignore.txt") as f:
-        return f.read().splitlines()
+    if pathlib.Path("data/ignore.txt").exists():
```

Comment:
Don't use external libraries for. something you can easily achieve with python.  Use try, except itself